### ISSUE TITLE
Deprecate zpages

### ISSUE THREAD

**Issue by cijothomas** [open-telemetry/opentelemetry-rust#1208]
Other languages have started deprecating zpages. The spec for it was experimental and has no plan to move to stable spec.
The total [download count ](https://crates.io/crates/opentelemetry-zpages) is <500 all time.

https://github.com/open-telemetry/opentelemetry-specification/pull/3577 Java
https://github.com/open-telemetry/opentelemetry-specification/pull/3564 .NET

**Comment by cijothomas** [comment]
https://github.com/open-telemetry/opentelemetry-rust/pull/1199 is disabling Zpages from CI check to unblock that PR. This issue is opened to track proper removal, along with deprecation notice in readme/crates.io etc

**Comment by TommyCpp** [comment]
Golang maintains a [zpage implementation](https://github.com/open-telemetry/opentelemetry-go-contrib/tree/main/zpages) so I think there may still be interest it in the future. Not a priority now but I am hesitate to delete it

**Comment by djc** [comment]
If there is interest in the future, it will be easy enough to resurrect from Git -- if no one really wants to spend the effort today to maintain it, we should just drop it. Once a new version of the opentelemetry crates comes out without a new version of the opentelemetry-zpages crate and someone notice, we'll know if someone wants it (enough to be willing to put in some work).

**Comment by hdost** [comment]
The reason that we were trying to disable to build for this moment is that we have to fix zpages before the build will complete due to some library changes.
I suppose we can put the effort into:
1. Fixing the build 
2. Add the deprecation/removal notice along with a note of if someone wants to maintain it to reach out in the project. 
3. Release the fixed version with the note 
4. Remove zpages from the repository with CHANGELOG note of the removal. 
That should be enough to close the ticket.
